docs/tutorials/mongoid-documents.txt

```diff
@@ -1147,6 +1147,109 @@ collection. An additional attribute ``_type`` is stored in order to make sure wh
 from the database the correct document is returned. This also holds true for the embedded
 documents ``Circle``, ``Rectangle``, and ``Shape``.
 
+Changing the Discriminator Key
+------------------------------
+Mongoid now supports changing the discriminator key from the default ``_type``. There are a few
```

Comment:
You can remove "now". We use the "Mongoid now supports ..." style in upgrade guide/release notes but not in the main body of documentation. Here we can just say "Mongoid supports ...".